_config.yml

```diff
@@ -6,13 +6,24 @@ url: "https://hyper.rs" # the base hostname & protocol for your site
 defaults:
   -
     scope:
-      path: "_guides"
+      path: "_legacy/"
       type: "guides"
     values:
       layout: "guide"
+  -
+    
+    scope:
+      path: "_stable/"
+      type: "guides"
+    values:
+      layout: "guide"
+  
 
 collections:
-  guides:
+  legacy:
+    permalink: /:collection/:path/
```

Comment:
Is it not possible to set the URL here? `/guides/0.14/:path/` and `/guides/1/:path`?

Reply:
It was indeed, thanks!